bezier-rs/lib/src/lib.rs

```diff
@@ -709,8 +730,8 @@ impl Bezier {
 				}
 			};
 
-			let min = curve.start.min(curve.end);
-			let max = curve.start.max(curve.end);
+			let min = other.start.min(other.end);
+			let max = other.start.max(other.end);
 
 			return list_intersection_t
```

Comment:
Remove `return`. Also a todo list item for you: if you can try and investigate why Clippy isn't automatically suggesting these things for you, that'd be a helpful at-the-time-of-writing code reviewer for you. The main codebase is set up to do that but I'm not sure why it isn't working here in your library.

Reply:
There `return` statement is necessary here because this is inside of an `if` block and instead having of an `else` block to handle the other case, I simply return early.

Reply:
Oh, my bad, somehow I didn't notice that this wasn't the end of the function. It looked like it when I was reading, and I even remember double checking. Strange, sorry!

Glad to hear Clippy is working also and that you're finding it useful.